README.md

```diff
@@ -42,44 +42,52 @@ from tidb_vector.sqlalchemy import VectorType
 engine = create_engine('mysql://****.root:******@gateway01.xxxxxx.shared.aws.tidbcloud.com:4000/test')
 Base = declarative_base()
 
-class Test(Base):
-    __tablename__ = 'test'
+class Document(Base):
+    __tablename__ = 'sqlalchemy_documents'
     id = Column(Integer, primary_key=True)
     embedding = Column(VectorType(3))
 
-# or add hnsw index when creating table
-class TestWithIndex(Base):
-    __tablename__ = 'test_with_index'
-    id = Column(Integer, primary_key=True)
-    embedding = Column(VectorType(3), comment="hnsw(distance=l2)")
-
 Base.metadata.create_all(engine)
 ```
 
 Insert vector data
 
 ```python
-test = Test(embedding=[1, 2, 3])
-session.add(test)
+doc = Document(embedding=[1, 2, 3])
+session.add(doc)
 session.commit()
 ```
 
 Get the nearest neighbors
 
 ```python
-session.scalars(select(Test).order_by(Test.embedding.l2_distance([1, 2, 3.1])).limit(5))
+session.scalars(select(Document).order_by(Document.embedding.l2_distance([1, 2, 3.1])).limit(5))
 ```
 
 Get the distance
 
 ```python
-session.scalars(select(Test.embedding.l2_distance([1, 2, 3.1])))
+session.scalars(select(Document.embedding.l2_distance([1, 2, 3.1])))
 ```
 
 Get within a certain distance
 
 ```python
-session.scalars(select(Test).filter(Test.embedding.l2_distance([1, 2, 3.1]) < 0.2))
+session.scalars(select(Document).filter(Document.embedding.l2_distance([1, 2, 3.1]) < 0.2))
+```
+
+Add hnsw index
+
+```python
+# vector index currently depends on tiflash
+session.execute(text('ALTER TABLE sqlalchemy_documents SET TIFLASH REPLICA 1'))
```

Comment:
Can this be encapsulated? User don't know what is TiFlash.

In the syntax design, an explicit SET TIFLASH REPLICA is intentional, because otherwise, adding a vector index **may** cause replicating a huge amount of data (because of implicit set replica 1) which is out of user expectation. However in this driver, the index is added right after table creation, thus there is no such risk.